compose.yml

```diff
@@ -17,6 +17,27 @@ services:
       - .:/app
     tty: true
     stdin_open: true
+  api:
+    depends_on:
+      - db
+    build: 
+      context: .
+      target: development
+      dockerfile: Dockerfile
+      args:
+        UID: ${UID:-1000}
+        GID: ${GID:-1000}
+        DEV: ${DEV:-false}
+        POETRY_VERSION: ${POETRY_VERSION:-1.8.4}
+    env_file:
+      - .env
+    volumes:
+      - .:/app
+    tty: true
+    stdin_open: true
+    command: ["poetry", "run", "fastapi", "dev", "dor/entrypoints/api/main.py", "--host", "0.0.0.0", "--port", "8000"]
+    ports:
+      - 8000:8000
```

Comment:
Can we parameterize this to `- ${API_PORT:-8000}:8000`?

Reply:
Probably, I'll take a look